File: dia-02/07-expressoes-aritmeticas/03-exercicio1-corrigido.cs
// 1. Exploração de operações matemáticas

using System;

class App
{
    static void Main()
    {
        // Solicita os números ao usuário
        Console.Write("Digite o primeiro número: ");
        string numeroDado1 = Console.ReadLine();
        double numero1 = double.Parse(numeroDado1); // ou Convert.ToDouble(numeroDado1)

        Console.Write("Digite o segundo número: ");
        string numeroDado2 = Console.ReadLine();
        double numero2 = double.Parse(numeroDado2);

        // Soma
        double soma = numero1 + numero2;
        Console.WriteLine($"\nSoma: {soma}");

        // Subtração
        double subtracao = numero1 - numero2;
        Console.WriteLine($"Subtração: {subtracao}");

        // Multiplicação
        double multiplicacao = numero1 * numero2;
        Console.WriteLine($"Multiplicação: {multiplicacao}");

        // Divisão
        // double divisao = numero1 / numero2; Se numero2 for 0, gera erro de runtime
        if (numero2 != 0)
        {
            double divisao = numero1 / numero2;
            Console.WriteLine($"Divisão: {divisao}");
        }
        else
        {
            Console.WriteLine("Divisão: impossível dividir por zero!");
        }

        // Resto
        // double resto = numero1 % numero2; Se numero2 for 0, gera erro de runtime
        if (numero2 != 0)
        {
            double resto = numero1 % numero2;
            Console.WriteLine($"Resto: {resto}");
        }
        else
        {
            Console.WriteLine("Resto: impossível calcular o módulo com divisor zero!");
        }
    }
}

File: MM5 Front/curso-adaptacao-js-edicao-2025/dia-01/04-expressoes-aritmeticas/03-exercicio1-corrigido.js
let num1 = 10;
let num2 = 5;

const soma = num1 + num2;
console.log("Soma: " + num1 + " + " + num2 + " = " + soma);

const subtracao = num1 - num2;
console.log("Subtração: " + num1 + " - " + num2 + " = " + subtracao);

const multiplicacao = num1 * num2;
console.log("Multiplicação: " + num1 + " * " + num2 + " = " + multiplicacao);

const divisao = num1 / num2;
console.log("Divisão: " + num1 + " / " + num2 + " = " + divisao);

const resto = num1 % num2;
console.log("Resto (Módulo): " + num1 + " % " + num2 + " = " + resto);

num2 = 0;
const resultadoDivisaoPorZero = num1 / num2;
console.log("Divisão por Zero: " + num1 + " / " + 0 + " = " + resultadoDivisaoPorZero);

num1 = 0;
const resultadoZeroPorZero = num1 / num2;
console.log("Divisão Zero por Zero: " + 0 + " / " + num1 + " = " + resultadoZeroPorZero);

